tensorflow/lite/g3doc/examples/image_classification/overview.md

```diff
@@ -110,7 +110,7 @@ Note: Image classification can only tell you the probability that an image
 represents one or more of the classes that the model was trained on. It cannot
 tell you the position or identity of objects within the image. If you need to
 identify objects and their positions within images, you should use an
-<a href="../object_detection/overview">object detection</a> model.
+<a href="https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/g3doc/examples/object_detection/overview.md">object detection</a> model.
```

Comment:
Is it sufficient to just add the ".md" extension, rather than also converting from a relative URL to an absolute URL?

If so, I think that would be preferable.

Reply:
This link works on the tensorflow doc site: https://www.tensorflow.org/lite/examples/image_classification/overview